nservicebus/messaging/routing.md

```diff
@@ -197,4 +187,4 @@ User-provided rules override the transport defaults. In order to register a rule
 
 snippet:Routing-TransportAddressRule
 
-A rule is a function that takes the instance name as the input and return a transport address or null if it does not provide translation for that particular instance name. All rules registered via this API have equal importance. It is expected that the user is responsible for providing a set of mutually exclusive rules so that for each endpoint instance name there is only one not-null translation result. In case there is more than one, an exception is raised.
\ No newline at end of file
+A rule is a function that takes the instance name as the input and return a transport address or null if it does not provide translation for that particular instance name. All rules registered via this API have equal importance. It is expected that the user is responsible for providing a set of mutually exclusive rules so that for each endpoint instance name there is only one not-null translation result. In case there is more than one, an exception is raised.
```

Comment:
This transport mapping will potentially go away if this one gets merged: https://github.com/Particular/NServiceBus/pull/3970